Server/src/config/envs/local.ts

```diff
@@ -81,6 +81,7 @@ export function createLocalConfig() {
         host: '0.0.0.0',
         port: 8055,
         generateFlows: false,
+        token: '-ssaT85n4S-wVD21LKOCDwvXN5PtnJc0',
```

Comment:
Is this token needed?

Reply:
Yes. This static token is used when core creats directus client. It is here: https://github.com/citrineos/citrineos-core/blob/823849c04e2a58212e9facff946496cdde218f5d/02_Util/src/util/directus.ts#L42 Then directus client works when, e.g., upload files. I updated it in docker config before but forgot local. 😞  https://github.com/citrineos/citrineos-core/blob/823849c04e2a58212e9facff946496cdde218f5d/Server/docker-compose.yml#L84